aclk/aclk_stats.c

```diff
@@ -269,10 +288,61 @@ static void aclk_stats_mqtt_wss(struct mqtt_wss_stats *stats)
         rd_recvd = rrddim_add(st, "received", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
     }
 
+    if (unlikely(!st_txbuf_perc)) {
+        st_txbuf_perc = rrdset_create_localhost(
+            "netdata", "aclk_mqtt_tx_perc", NULL, "aclk", NULL, "Activelly used percentage of MQTT Tx Buffer,", "%",
```

Comment:
Only small typo, I think it should be `Actively` ?

Reply:
or Actually?

Reply:
Thanks, I think actively is better as actual used part of buffer is more than this chart shows as beginning of buffer is used by data marked for garbage collection. To decrease CPU usage and memory moves we do the garbage collection only at the time when we don't have space for new message metadata at the end of the buffer (so we shift it left and remove the garbage collectible items).

Reply:
thx, fixed the typo

Reply:
> or Actually?
> Thanks, I think actively is better as actual used part

It was a joke! I know, I know it's almost impossible to notice when there are no emojis 😄

Reply:
:D clearly the constant 30+ DEGC weather in asia makes my CPU work with lower clocks :D